spec/status.xml

```diff
@@ -5,62 +5,47 @@
+<bibref ref="ttml10"/> clarifies ambiguities and corrects errors identified in the 
+<loc href="&prevrec;">previous
+edition</loc>, which this new edition supersedes. No features were added.</p>
 
-<p>This document is a revision of Timed Text Markup Language (TTML) 1.0
-<bibref ref="ttml10"/>, superseding that <loc href="http://www.w3.org/TR/2010/REC-ttaf1-dfxp-20101118/">previous
-Recommendation</loc>. The only identified substantive difference between this version and
-its predecessor is a change of the default treatment of the <att>ttp:markerMode</att> parameter attribute
-from <code>continuous</code> to <code>discontinuous</code>. A cumulative summary of all changes applied
-to this version since the Recommendation was first published is available at
-<loc  href="ttml1-changes.html#change-history-rec-to-2e-per">Changes
-from Recommendation to Proposed Edited Recommendation (2nd Ed.)</loc> and
-<loc  href="ttml1-changes.html#change-history-2e-per-to-rec-2e">Changes
-from Proposed Edited Recommendation (2nd Ed.) to Recommendation (2nd Ed.)</loc>.</p>
+<p>Substantive changes applied since the <loc href="&prevloc;">previous version of this specification</loc> are listed at 
+<loc href="substantive-changes-summary.txt">substantive-changes-summary.txt</loc>.</p>
 
-<p>A <loc href="http://www.w3.org/2008/10/dfxp-testsuite.zip">sample test suite</loc> is available. This set of tests is not intended to
-be exhaustive. The <loc href="http://www.w3.org/2009/05/dfxp-results.html">implementation report</loc> used by the director to
-transition to Recommendation has been made available. There have been no formal objections
-to the publication of this document.</p>
+<p>For convenience, a diff between this and the <loc href="&prevrec;">previous edition</loc> 
+is offered at the <loc href="https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FTR%2F2013%2FREC-ttml1-20130924%2F%26doc2=https%3A%2F%2Fwww.w3.org%2FTR%2F2018%2FCR-ttml1-20180313%2F">W3C HTML Diff service</loc>.</p>
 
-<p>This document was produced by the <loc
-href="http://www.w3.org/AudioVideo/TT/">Timed Text (TT) Working Group</loc> as
-part of the W3C <loc href="http://www.w3.org/2008/WebVideo/">Video
-in the Web Activity</loc>, following the procedures set out for the W3C <loc
-href="http://www.w3.org/Consortium/Process/">Process</loc>.  The authors of this
-document are listed in the header of this document, including input and review
-by liaisons from the <loc href="http://www3.ebu.ch/cms/en/home">European
-Broadcasting Union (EBU)</loc>, and the <loc href="https://www.smpte.org">Society
-of Motion Picture &amp; Television Engineers (SMPTE)</loc>.</p>
+<p>For this specification to exit the CR stage, at least 2 independent implementations of every <emph>CR exit criteria test</emph> is required, as documented in the Working Group's <loc href="">implementation report</loc>. The Working Group does not require that implementations are publicly available but encourages them to be so.</p>
```

Comment:
Looking at `spec/substantive-changes-summary.txt`, I'm not sure if the test suite is likely to contain anything. If it does not, then we will need to change this sentence to state that, since no features are being introduced, the CR exit criteria do not include any dependencies on implementations. _OR_ are we going to create tests to check that the clarified behaviours are implemented as intended? I'm not sure what we will need to do here.

Reply:
I've sent a message to @plehegar about this - just bringing it to your attention in context Philippe.